client/src/components/pages/application/Profile/index.js

```diff
@@ -1,5 +1,329 @@
+                    <tr>
+                      <td className="item" colSpan="2">
+                        <Typography variant="body2" align="left">
+                          <Checkbox
+                            checked={data && data.technicalTasks}
+                            className="profile_check_box"
+                            color="primary"
+                          />
+                          {taskName}
+                        </Typography>
+                      </td>
+                    </tr>
+                  ))}
+                <tr className="doted">
+                  <td>
+                    <Typography variant="body2" color="primary" align="left">
+                      Project
+                    </Typography>
+                  </td>
+                  <td />
+                </tr>
+                <tr>
+                  <td className="item" colSpan="2">
+                    <Typography variant="body2" align="left">
+                      <Checkbox
+                        checked={data && data.projectGithubLink}
+                        className="profile_check_box"
+                        color="primary"
+                      />
+                      <a
+                        className="Link"
+                        href={data.projectGithubLink}
+                        target="popup"
+                        onClick={() => {
+                          window.open(
+                            'https://discord.com/',
+                            'popup',
+                            'width=600,height=600,scrollbars=no,resizable=no'
+                          );
+                        }}
+                      >
+                        Project Github Link
+                      </a>
+                    </Typography>
+                  </td>
+                </tr>
+                <tr className="doted">
+                  <td>
+                    <Typography variant="body2" color="primary" align="left">
+                      Submitted
+                    </Typography>
+                  </td>
+                  <td />
+                </tr>
+                <tr>
+                  <td className="item">
+                    <Typography variant="body2" align="left">
+                      Code Wares Score
+                    </Typography>
+                  </td>
+                  <td>
+                    <Typography variant="body2" color="primary" align="left">
+                      {data.codeWarsKyu}
+                    </Typography>
+                  </td>
+                </tr>
+                <tr>
+                  <td className="item">
+                    <Typography variant="body2" align="left">
+                      FreeCode Camp Score
+                    </Typography>
+                  </td>
+                  <td>
+                    <Typography variant="body2" color="primary" align="left">
+                      {data.freeCodeCampPoints}
```

Comment:
numbers appears a bit lower than its name, work on its style, the same for codewars